backend/app/controllers/spree/admin/taxons_controller.rb

```diff
@@ -72,6 +72,19 @@ def destroy
         respond_with(@taxon) { |format| format.json { render json: '' } }
       end
 
+      def clear_icon
+        @taxon = Spree::Taxon.find(params[:taxon_id])
+        @taxon.icon = nil
```

Comment:
In order to save one line of code this could be written also like this:
 
```ruby
if @taxon.update(icon: nil)
  flash[:success] = flash_message_for(@taxon, :successfully_icon_cleared)
end